frequency.rb

```diff
@@ -0,0 +1,14 @@
+#!/urs/bin/ruby -w
+
+hash = {}
+str = String.new("aabbbcddeac")
```

Comment:
You can directly write
```suggestion
str = 'aabbbcddeac'
```